abstract_turtle/turtle_class.py

```diff
@@ -64,13 +64,15 @@ def setheading(self, heading):
     seth = setheading
 
     @turtle_method
-    def circle(self, radius, extent=360):
+    def circle(self, radius, extent=None):
         """
         Draw a circle starting at the given point with the given RADIUS and EXTENT. If EXTENT exists, draw only the
         first EXTENT degrees of the circle. If RADIUS is positive, draw in the counterclockwise direction.
         Otherwise, draw in the clockwise direction.
         """
-
+        if extent is None:
+            extent = 360
```

Comment:
I think this is what you meant right?
```suggestion
            extent = self.__degrees
```

Reply:
Oh yeah probably.